linux/just_files/new_just

```diff
@@ -474,97 +479,117 @@ function write_dockerfile()
 {
   exists "${1}" && return 0
 
-  uwecho   'FROM vsiri/recipe:gosu as gosu
+  uwecho   '#T# Dockers use gosu instead of sudo for smoother operation
```

Comment:
The just entrypoint does that. _Real_ `sudo` is not optimal for docker, because it uses tty for interactive input to get a password, and that would break in a number of situations, e.g. `docker build`. The alias `sudo` has nothing to do with the _real_ sudo. The alias `sudo` is there because you complained, not so you can say "sudo is there", cause it's not. Dockers use gosu instead of sudo, for smoother operation